op-e2e/config/init.go

```diff
@@ -409,14 +413,11 @@ func defaultIntent(root string, loc *artifacts.Locator, deployer common.Address,
 							DisputeAbsolutePrestate: defaultPrestate,
 							DisputeMaxGameDepth:     14 + 3 + 1,
 							DisputeSplitDepth:       14,
-							DisputeClockExtension:   0,
-							DisputeMaxClockDuration: 0,
+							DisputeClockExtension:   8,
+							DisputeMaxClockDuration: 200,
```

Comment:
Could we make this 12 to speed up withdrawal tests that use the fast game.
```suggestion
							DisputeMaxClockDuration: 12,
```

Reply:
followed up offline. We'll leave this at 120 instead.

Reply:
done